py3status/modules/window_title.py

```diff
@@ -51,7 +51,12 @@ def __init__(self):
     def window_title(self):
         window = find_focused(i3.get_tree())
 
+        if not window or window.get('name') is None or \
+           str(window.get('num')) == window.get('name'):
```

Comment:
It doesn't happen in window_title_async.

In my opinion, it is a bug, because empty workspace doesn't have active window and it should not show a workspace number as window name.